src/settings/OrderNumber.js

```diff
@@ -22,44 +14,61 @@ class OrderNumber extends Component {
 
   constructor(props) {
     super(props);
+
     this.configManager = props.stripes.connect(ConfigManager);
   }
 
   getInitialValues = (settings) => {
-    const value = settings.length && settings[0].value === 'true';
+    const value = settings.length === 0 ? '' : settings[0].value;
+    const defaultConfig = {
```

Comment:
if you create it here, just name it after the setting - `config`. You can extend it later with values from back-end